.gitignore

```diff
@@ -11,3 +11,8 @@
 
 # Generated test specs
 /tests/moonbeam-test-specs/*.json
+
+# Moonbeam-Launch
+*.log
+rococo-local.json
```

Comment:
Not needed I believe

Reply:
those specs are generated by the launch script

Reply:
Can we make it so it is generated in a folder that we ignore then?

Reply:
sure